Modules/Core/Common/include/itkMultiThreaderBase.h

```diff
@@ -243,137 +263,133 @@ ITK_GCC_PRAGMA_DIAG_POP()
    * This implementation simply delegates parallelization to the old interface
    * SetSingleMethod+SingleMethodExecute. This method is meant to be overloaded! */
   virtual void
-  ParallelizeArray(
-    SizeValueType firstIndex,
-    SizeValueType lastIndexPlus1,
-    ArrayThreadingFunctorType aFunc,
-    ProcessObject* filter );
+  ParallelizeArray( SizeValueType firstIndex, SizeValueType lastIndexPlus1, ArrayThreadingFunctorType aFunc,
+                    ProcessObject * filter );
 
   /** Break up region into smaller chunks, and call the function with chunks as parameters.
    * If filter argument is not nullptr, this function will update its progress
    * as each work unit is completed. Delegates work to non-templated version. */
-  template<unsigned int VDimension>
-  ITK_TEMPLATE_EXPORT void ParallelizeImageRegion(const ImageRegion<VDimension> & requestedRegion,
-                                     TemplatedThreadingFunctorType<VDimension> funcP,
-                                     ProcessObject* filter)
+  template < unsigned int VDimension >
+  ITK_TEMPLATE_EXPORT void
+  ParallelizeImageRegion( const ImageRegion< VDimension > &           requestedRegion,
```

Comment:
This alignment of parameter names is weird, and perhaps counterproductive. I guess there is no setting which says: if parameters cannot all fit on a single line, make each parameter have its own line? If there is, can we have that please?

Reply:
There is!

`
AllowAllParametersOfDeclarationOnNextLine (bool)
If the function declaration doesn’t fit on a line, allow putting all parameters of a function declaration onto the next line even if BinPackParameters is false.
`